api/stub/faults.go

```diff
@@ -0,0 +1,30 @@
+package stub
```

Comment:
We can probably get rid of all this stub code now that we have a reliable Devnet setup. No big deal either way I guess.